sdk/core/System.ClientModel/src/Convenience/OperationResult.cs

```diff
@@ -0,0 +1,79 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System.Threading.Tasks;
+
+namespace System.ClientModel.Primitives;
+
+/// <summary>
+/// Represents an operation that runs asynchronously on a cloud service.  Cloud
+/// services use long-running operations to allow users to start an operation
+/// with one request and then monitor progress of the operation until it has
+/// completed.  <see cref="OperationResult"/> enables waiting for completion of
+/// long-running operations.  Client libraries provide derived types that add
+/// properties such as <code>Value</code> or <code>Status</code> as applicable
+/// for a given service operation.
+/// </summary>
+public abstract class OperationResult : ClientResult
+{
+    /// <summary>
+    /// Create a new instance of <see cref="OperationResult"/>.
+    /// </summary>
+    /// <param name="response">The <see cref="PipelineResponse"/> received from
+    /// the service in response to the request that started the operation.</param>
+    /// <remarks>Derived types will call
+    /// <see cref="ClientResult.SetRawResponse(PipelineResponse)"/> when a new
+    /// response is received that updates the status of the operation.</remarks>
+    protected OperationResult(PipelineResponse response)
+        : base(response)
+    {
+    }
+
+    /// <summary>
+    /// Gets a value that indicates whether the operation has completed.
+    /// </summary>
+    /// <value>`true` if the operation has completed (that is, the service is
+    /// done processing the operation and it has terminated due to having
+    /// finished successfully, because of an error condition, or having been
+    /// cancelled by a user); otherwise, `false`.
```

Comment:
just a suggestion to make it a bit more concise. Feel free to ignore.

```suggestion
    /// <value>`true` if the operation has reached a terminal state (that is, it has finished
    /// successfully, reached an error condition, or has been cancelled by a user);
    /// otherwise, `false`.
```

Reply:
Good edit, and thank you!